### ISSUE TITLE
'use of undefined value' with JIT

### ISSUE THREAD

**Issue by november-kilo** [dworkin/lpc-ext#3]
I see an error message in the DGD console using latest from dgd and lpc-ext repositories (only JIT extension is used) and my cloud-server fork (also occurs in base cloud-server repository with the same changes) using my mac. The message appears only once with a new cache. 

I'm unable to try on another computer for a while.

Procedure:
- create `cache/` in jit extension
- boot mud
- log in as admin, grant a user access
- log in as that user
- type `foo`
- message in DGD console
```
cache/ca/ca5af742ad29b8c4074b06877db045b8.ll:157:41: error: use of undefined value '%t10'
        %t12 = call double %c13(i8* %f, double %t10, double %t11)
                                               ^
1 error generated.
```

Files:
- https://github.com/november-kilo/cloud-server/blob/master/src/include/Foo.h
- https://github.com/november-kilo/cloud-server/blob/master/src/lib/Foo.c
- https://github.com/november-kilo/cloud-server/blob/master/src/usr/System/initd.c (line 62)
- https://github.com/november-kilo/cloud-server/blob/master/src/usr/System/obj/user.c (lines 9, 608, 955-957)

Compiler:
```
$ uname -s
Darwin
$ which cc
/usr/bin/cc
$ cc -v
Apple LLVM version 10.0.1 (clang-1001.0.46.4)
Target: x86_64-apple-darwin18.7.0
Thread model: posix
InstalledDir: /Library/Developer/CommandLineTools/usr/bin
$ which g++
/usr/bin/g++
$ g++ -v
Configured with: --prefix=/Library/Developer/CommandLineTools/usr --with-gxx-include-dir=/usr/include/c++/4.2.1
Apple LLVM version 10.0.1 (clang-1001.0.46.4)
Target: x86_64-apple-darwin18.7.0
Thread model: posix
InstalledDir: /Library/Developer/CommandLineTools/usr/bin
```

**Comment by dworkin** [comment]
Haven't been able to reproduce it exactly yet, but another error popped up that could be caused by a JIT compiler problem.

It seems like you are using macOS 10.14 but with older Xcode?

**Comment by dworkin** [comment]
Reproduced on 10.13 with Xcode 10.1.

**Comment by november-kilo** [comment]
I'm using macOS version 10.14.6

I use the Xcode standalone command line developer tools. I removed them and installed with `xcode-select --install`. I get xcode-select version 2354, which is the same as before, and the same results after recompiling DGD and LPC-ext. Perhaps it doesn't install the correct command line tools for this mac.

I just now installed Xcode 11.3.1 but haven't figured out how to use it yet.
